### ISSUE TITLE
Incorrect documentation

### ISSUE THREAD

**Issue by iofit** [revelc/formatter-maven-plugin#882]
**Describe the bug**
In documentation for [external configuration resource](https://code.revelc.net/formatter-maven-plugin/examples.html), `<configFile>` is inside `<execution>` element, it took me a couple hours of debugging to realize this is not correct, plugin won't read anything inside `<execution>` for `<configuration>` elements, `<configuration>` has to be always below <plugin> not <execution>.

**Versions (OS, Maven, Java, and others, as appropriate):**
 - Affected version(s) of this project: 2.23.0
 - OS: Windows 11
 - Others: 

**To Reproduce**
Steps to reproduce the behavior (or a link to an example repository that reproduces the problem):
1. Go to https://code.revelc.net/formatter-maven-plugin/examples.html
2. Check example source under "The following configuration will bind the plugin to execute the format goal in the default lifecycle phase of process-sources. Try it using mvn process-sources"

**Expected behavior**
Documentation should be accurate so when plugin is used it works.

**Screenshots**
NA

**Additional context**
NA

**Comment by hazendaz** [comment]
Not sure I follow.  I see 1, 2, but what am I scrolling down to as ... there?  The context at 2 is accurate. You say there is an error, what error?

**Comment by iofit** [comment]
<configuration> item should be under <plugin>, in the section "Basic Configuration Using External Resource", the second code snippet shows <configuration> under <executions> which from my debug won't take effect:

![image](https://github.com/revelc/formatter-maven-plugin/assets/78842/42beecb4-2cc4-4395-9fee-963ff6a20e8d)

**Comment by hazendaz** [comment]
It works under both.  That is how maven works.  If at plugin level configuration, it works on all executions of the plugin.  If under a specific execution, it will only work for that execution.

Sent from my Verizon, Samsung Galaxy smartphone
Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by ctubbsii** [comment]
Part of this is hard to follow because your original post included `<` and `>` characters that were not escaped. Please remember that posts on GitHub use Markdown, so anything you put in a comment or post that looks like an XML element will get hidden. I'll update the original post to add the appropriate escapes.

**Comment by ctubbsii** [comment]
I agree with @hazendaz that it works under both. The plugin doesn't parse the configuration from the POM. That is done by Maven itself, and the plugin merely receives the configuration parsed by Maven. You can place configuration in several places: the command-line as properties, in a parent POM, in the pluginManagement section under the plugin, in the normal plugins section under the plugin, inside a specific execution, inside a profile, etc. How all this gets resolved and merged together depends on Maven.

It would help if you provided the configuration that did not work for you, and then we might be able to help troubleshoot why it didn't work for you. But as-is, this does not seem like a valid issue, so I'm closing it.
